static/app/views/releases/list/releaseListDropdown.tsx

```diff
@@ -1,13 +1,19 @@
+import {ComponentProps} from 'react';
+
 import DropdownControl, {DropdownItem} from 'app/components/dropdownControl';
+import Tooltip from 'app/components/tooltip';
 
-import {DisplayOption, SortOption, StatusOption} from './utils';
+type DropdownItemProps = Pick<
+  ComponentProps<typeof DropdownItem>,
+  'disabled' | 'title'
+> & {
+  label: string;
+  tooltip?: string;
+};
 
 type Props = {
   label: string;
-  options:
-    | Record<DisplayOption, string>
-    | Record<SortOption, string>
-    | Record<StatusOption, string>;
+  options: Record<string, DropdownItemProps>;
```

Comment:
this was somewhat unfortunate, but i needed more than just a label for each option

Reply:
i could also not reuse this component for ReleaseListSortOptions